pkgs/development/compilers/ghc/head.nix

```
@@ -12,9 +12,10 @@
   libffi ? null
 
   # Libdw.c only supports x86_64, i686 and s390x
-, enableDwarf ? stdenv.targetPlatform.isx86 &&
-                !stdenv.targetPlatform.isDarwin &&
-                !stdenv.targetPlatform.isWindows
+, enableDwarf ? (stdenv.targetPlatform.isx86 ||
+                 stdenv.targetPlatform.isS390) &&
```

Sorry for another roundtrip, but I think we need to check

```suggestion
                 stdenv.targetPlatform.parsed.cpu.name == "s390x") &&
```

instead, since `isS390` is true for `s390` and `s390x`, but Libdw.c only explicitly supports `s390x`. Wdyt?